helm_deploy/values-dev.yaml

```
@@ -15,6 +15,7 @@ generic-service:
 
   allowlist:
     capita-dev: 51.104.16.30/31
+    neil-test: 51.155.102.238
```

I think it needs to be a CIDR rather than an IP, but have approved it anyway if you want to experiment

```suggestion
    neil-test: 51.155.102.238/32
```